### ISSUE TITLE
Setup continuous integration for the web service Python project

### ISSUE THREAD

**Issue by naitoarekkus** [P7-Team/P7-WebService#9]
**Is your feature request related to a problem? Please describe.**
It is related to issue #8 , in that issue 8 needs to be finished before start.

**Describe the solution you'd like**
A continuous integration workflow should be setup for the web service Python project.
This should automatically run all automatic tests and check that they all pass, before they will allow a pull request to be merged into the main/develop branch.

**Describe alternatives you've considered**
Github actions could be a way to set it up

**Additional context**
Something might already be configured for in github workflow
